bin/qds.py

```diff
@@ -113,10 +114,10 @@ def submitaction(cmdclass, args):
         return 0
 
 
-def _getresult(cmdclass, cmd):
+def _getresult(cmdclass, cmd, args=[]):
```

Comment:
Looked into it.We can't pass hash here as its a single command line argument although it can be converted to hash internally but then we have create new function by converting single line arguments to hash . We generally use hash where we we use optparser or argumentparser and in return we get hash which is not the case here.